variables.tf

```diff
@@ -1036,3 +1036,26 @@ variable "saml-x509-keypair" {
   default     = ""
 }
 
+variable "enable-cloudkitty" {
+  description = "Enable Openstack Rating service"
+  type        = bool
+  default     = false
+}
+
+variable "cloudkitty-channel" {
+  description = "Operator channel for Cloudkitty deployment"
+  type        = string
+  default     = null
```

Comment:
Since we are not supporting 2024.1 for cloudkitty and openstack-channel default to 2024.1, can we set default for cloudkitty-channel to 2025.1/edge for now